api_tests/wb/views/test_wb_hooks.py

```diff
@@ -802,6 +840,43 @@ def test_invalid_payload(self, app, copy_url):
         assert res.status_code == 400
         assert res.json['errors'][0]['detail'] == 'Invalid Payload'
 
+    def test_storage_usage_move_within_node(self, app, node, signed_payload, copy_url):
+        """
+        Checking copys within a node, since the net size of the node hasn't changed the cache will remain expired at None
+        """
+        assert node.storage_usage is None
+
+        res = app.post_json(copy_url, signed_payload)
+
+        assert res.status_code == 201
+        assert node.storage_usage is None  # this is intentional the cache shouldn't be touched
```

Comment:
This doesn't make sense to me, we are treating copies as "literal copies" when calculating storage. If you copy a file in the same folder, the net size *will* change.

Reply:
Yes you're 100% correct, I just got mixed up.